src/sidebars/favourites/tearout-directive.js

```diff
@@ -137,7 +137,6 @@
                                         }
 
                                         // Remove drop-target from original instance
-                                        parent.removeChild(myHoverArea);
```

Comment:
The equivalent of this would be `parent.remove()`, but since everything's removed anyway on tear out I didn't think it worth it